packages/rum/src/domain/record/record.spec.ts

```diff
@@ -363,12 +363,37 @@ describe('record', () => {
       expect(mutationData.removes.length).toBe(1)
     })
 
-    function createShadow(children: Element[]) {
+    it('should clean the state when both the parent  shadow host is removed to avoid memory leak', () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I think you're missing a "and the" in the title